core/src/main/java/org/apache/iceberg/util/PropertyUtil.java

```diff
@@ -36,7 +37,7 @@ public static boolean propertyAsBoolean(Map<String, String> properties,
   }
 
   public static double propertyAsDouble(Map<String, String> properties,
-      String property, double defaultValue) {
+                                        String property, double defaultValue) {
```

Comment:
Nit - Unnecessary change in this PR. Ideally, we keep every line change specific to this one PR so that groups that might maintain a fork have an easier time cherry-picking what they need etc.

Reply:
It's because of my IDE (IntelliJ). I changed it back.
Could you take a look